hardware/raspberrypi/bootmodes/bootflow_2711.md

```diff
@@ -1,31 +1,67 @@
-# Pi4 Bootflow
+# Raspberry Pi4, Pi400 and CM4 bootflow
 
-The Pi4 with the BCM2711 SoC has a new, more sophisticated boot process. The addition of an EEPROM means that the `bootcode.bin` file found in `/boot` is no longer required. Details of the EEPROM can be found [here](../booteeprom.md).
+This page describes the bootflow for BCM2711 based products. The main difference betweeen this and previous products is that the second stage bootloader (bootcode.bin) is loaded from an SPI flash [EEPROM](../booteeprom.md). 
 
-The boot flow for the Pi4 is as follows:
+The bootflow for the ROM (first stage) is as follows:-
 
 * BCM2711 SoC powers up
-* On board bootrom checks for bootloader recovery file (recovery.bin) on the SD card. If found, it executes it to flash the EEPROM and recovery.bin triggers a reset.
-* Otherwise, the bootrom loads the main bootloader from the EEPROM.
-* Bootloader checks it's inbuilt BOOT_ORDER configuration item to determine what type of boot to do.
-  * SD Card
-  * Network
-  * USB mass storage
+* Read OTP to determine if board model defined `nRPIBOOT` GPIO
```

Comment:
I dunno what "board model defined" means here - could the whole line be simplified to just "Read OTP to determine if  `nRPIBOOT` GPIO is defined" ?
(and then should the "N.B. Currently only CM4 reserves a GPIO for `nRPIBOOT`" be changed to N.B. "Currently only CM4 defines a GPIO for `nRPIBOOT`" ?)

Reply:
reworded